ArgoProject/src/systems/InputSystem.cpp

```diff
@@ -14,21 +16,26 @@ app::sys::InputSystem::InputSystem()
 void app::sys::InputSystem::update(app::time::seconds const & dt)
 {
 
-	m_registry.view<comp::Input, comp::Commandable>()
-		.each([&, this](app::Entity const entity, comp::Input & input, comp::Commandable& commandable) 
+	m_registry.view<comp::Input, comp::Commandable, comp::Audio>()
```

Comment:
```suggestion
	m_registry.view<comp::Input, comp::Commandable>()
```
There is a better way, I think I have better solution, discuss in person.